docs/getting-started-guides/scratch.md

```diff
@@ -66,21 +67,25 @@ the node is added.  A process in one pod should be able to communicate with
 another pod using the IP of the second pod.  This connectivity can be
 accomplished in two ways:
 
-- Configure network to route Pod IPs
+- Configure underlay network to route Pod IPs
```

Comment:
As this is one of the two major methods, I'd format it as bold text so that it's more obvious.